newdle/client/src/components/RecipientList.js

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Label, List} from 'semantic-ui-react';
+import styles from './ParticipantTable.module.scss';
+
+const DEFAULT_RECIPIENTS_SHOWN = 10;
+
+function ListItems({recipients, color, icon}) {
+  return (
+    <>
+      <List>
+        {recipients.map(p => {
+          return (
+            <List.Item key={p.id}>
+              <List.Icon color={color} name={icon} />
+              <List.Content>{p.name}</List.Content>
+            </List.Item>
+          );
+        })}
+      </List>
+    </>
+  );
+}
+
+ListItems.propTypes = {
+  recipients: PropTypes.arrayOf(
+    PropTypes.shape({id: PropTypes.number.isRequired, name: PropTypes.string.isRequired})
```

Comment:
I thought prettier would take care of it but it didn't :(

```suggestion
    PropTypes.shape({
      id: PropTypes.number.isRequired,
      name: PropTypes.string.isRequired,
    })
```